ragna/source_storages/_chroma.py

```diff
@@ -33,58 +39,59 @@ def __init__(self) -> None:
             )
         )
 
+        self._tokens = 0
+        self._embeddings = 0
```

Comment:
These will change with every `.store` call. Why are they instance attributes rather than local variables?

Reply:
These variables represent the average length of a chunk within the source storage, It is aggregated across calls to store.  I'm not sure what scope you're referring to but I don't think they can be local for them to work.